src/Features/CSharp/Portable/xlf/CSharpFeaturesResources.zh-Hant.xlf

```diff
@@ -122,6 +122,11 @@
         <target state="translated">排序協助工具修飾詞</target>
         <note />
       </trans-unit>
+      <trans-unit id="Use_asynchronous_statement">
+        <source>Use asynchronous statement</source>
+        <target state="new">Use asynchronous statement</target>
```

Comment:
We already have `Make method async`.  Consider `Make async` or `Make statement async`.

Reply:
N/m.  You're adding 'await'.  Consider just reusing the `Add_await` resource we already have.